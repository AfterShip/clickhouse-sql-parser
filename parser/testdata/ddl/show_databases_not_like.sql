SHOW DATABASES NOT LIKE 'temp%'