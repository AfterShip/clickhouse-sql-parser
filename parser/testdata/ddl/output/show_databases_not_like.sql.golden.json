[
  {
    "ShowPos": 0,
    "StatementEnd": 32,
    "ShowType": "DATABASES",
    "Target": null,
    "NotLike": true,
    "LikeType": "LIKE",
    "LikePattern": {
      "LiteralPos": 25,
      "LiteralEnd": 30,
      "Literal": "temp%"
    },
    "Limit": null,
    "OutFile": null,
    "Format": null
  }
]